jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/ServletApiRequest.java

```diff
@@ -1209,8 +1209,15 @@ public ServletInputStream getInputStream() throws IOException
         if (_inputState != ServletContextRequest.INPUT_NONE && _inputState != ServletContextRequest.INPUT_STREAM)
             throw new IllegalStateException("READER");
         _inputState = ServletContextRequest.INPUT_STREAM;
-        // Try to write a 100 continue, ignoring failure result if it was not necessary.
-        _servletChannel.getResponse().writeInterim(HttpStatus.CONTINUE_100, HttpFields.EMPTY);
+        try
+        {
+            // Try to write a 100 continue, ignoring failure result if it was not necessary.
+            _servletChannel.getResponse().writeInterim(HttpStatus.CONTINUE_100, HttpFields.EMPTY);
+        }
+        catch (IllegalStateException ise)
```

Comment:
this kind of conflicts with the comment above? please explain?

Reply:
You are right, the `IllegalStateException` should be ignored.